src/store/todo/todo.actions.ts

```diff
@@ -0,0 +1,72 @@
+import { createAction } from '@reduxjs/toolkit';
+import { TodoDTO, TodosColumns } from '~types/todo.types';
+
+const todoAddRequested = createAction<TodoDTO>('todoAdd/requested');
```

Comment:
Let's place each action type in it's own file and then merge them in one index file, I mean like this
`todoAdd.action.ts -> export const todoAddRequested = createAction<TodoDTO>('todoAdd/requested');`

Reply:
After rewriting with reduxSlice, actions are generated automatically, so there are no need in using createAction there anymore, thus no need in actions files, so deleted them, but actions still are exported via index.ts